src/main/kotlin/no/nav/syfo/sykmelding/util/JaxBUtils.kt

```diff
@@ -0,0 +1,26 @@
+package no.nav.syfo.sykmelding.util
+
+import com.migesok.jaxb.adapter.javatime.LocalDateTimeXmlAdapter
+import com.migesok.jaxb.adapter.javatime.LocalDateXmlAdapter
+import java.io.StringWriter
+import javax.xml.bind.JAXBContext
+import javax.xml.bind.Marshaller
+import javax.xml.bind.Marshaller.JAXB_ENCODING
+import javax.xml.bind.Unmarshaller
+import no.nav.helse.eiFellesformat.XMLEIFellesformat
+import no.nav.helse.msgHead.XMLMsgHead
+import no.nav.helse.sm2013.HelseOpplysningerArbeidsuforhet
+
+val fellesformatJaxBContext: JAXBContext = JAXBContext.newInstance(XMLEIFellesformat::class.java, XMLMsgHead::class.java, HelseOpplysningerArbeidsuforhet::class.java)
```

Comment:
Hvordan blir dette her med tanke på trådsikkerhet..? @andreasDev , du er god på sånt, hva tror du? :)

Reply:
I påvente av at Andreas svarer selv: Jeg hadde opprinnelig den same copypastaen i min branch, men fjernet det da du tok over XML-biten. Andreas kommenterte da på min PR at det neppe var trådsikkert og at vi kanskje ikke burde gjøre det slik. Kanskje tryggere å lage en egen instans hver gang det brukes?

Reply:
Det kan jeg gjøre :) Blir litt overhead pr kall, men bedre enn å blande sammen sykmeldingene og skrive en salig røre til topic 🙈

Reply:
Undersøkte litt, og JAXB-contexten er visstnok trådsikker, mens marshaller/unmarshaller IKKE er trådsikre. 

Den hjelpemetoden for toString() på marshalleren er kanskje grei sånn den står, eller kan den potensielt være skummel også?

Reply:
Den tar jo imot en input som den jobber på, så den *burde* jo være trygg... men gudene vet. Et alternativ hadde vel kanskje vært å bare lage objekter av JaxBUtils hver gang den trengtes. Det er jo et overhead med å instansiere per kall som du sier, men bare konstant tid O(1), og det kan vi jo bare se bort ifra :p

Reply:
Alternativt kan du bare merge det inn med en TODO der, så kan Andreas og Joakim få komme med sine tanker senere. Det er litt tid til vi går i prod uansett :)